src/main/java/org/apache/mesos/offer/TaskUtils.java

```diff
@@ -309,12 +310,49 @@ public static String getTaskType(TaskInfo taskInfo) throws InvalidProtocolBuffer
         }
     }
 
-    public static CommandInfo getCommand(TaskInfo taskInfo) throws InvalidProtocolBufferException {
-        return Protos.CommandInfo.parseFrom(taskInfo.getData());
+    public static TaskInfo serializeTaskInfo(TaskInfo taskInfo) {
```

Comment:
Maybe slightly more explanatory names for this method pair? It's not the `TaskInfo` being serialized, but the attached `CommandInfo`.